features/Proxy.feature

```diff
@@ -207,3 +207,40 @@ Feature: Proxy requests to upstream.
 
         When I request "/posts-public/nested/path" path with "GET" method
         Then I should receive 404 response code
+
+        Given request JSON payload:
+            """
+            {
+                "name":"parameters_interpolation",
+                "active":true,
+                "proxy":{
+                    "preserve_host":false,
+                    "listen_path":"/api/recipes/{id:[\\da-f]{24}}",
```

Comment:
Could we also test `/api/recipes/{id:[\\da-f]{24}}/menus/{id:[\\da-f]{24}}` to ensure it also works with more than one parameter?

Reply:
sure, I'll add it, it should be then
`/api/recipes/{recipeId:[\\da-f]{24}}/menus/{menuId:[\\da-f]{24}}`